### ISSUE TITLE
How i can find bot-config.json?

### ISSUE THREAD

**Issue by YukaNozomi** [Malexion/Rhythm-Bot#27]
I looking in all file but not found it.Could you help me?

**Comment by Malexion** [comment]
Sure in the main folder you'll want to create a bot-config.json file if it does not already exist. Running the application once will also create it.

Contents of the file look as follows, and you replace everything between " quotes with your token.

```json
{
    "discord": {
        "token": "token-here"
    }
}
```

Token comes from the discord developer portal after you make an "app" here:

![Where to get the token](https://user-images.githubusercontent.com/5237473/90321068-071a7f00-df0c-11ea-9c60-3154819a7f94.png)

**Comment by NovaPM** [comment]
I ran the program but still can't find the file :/

**Comment by camden-git** [comment]
Should you close this issue as it was resolved?

**Comment by mattzaharias** [comment]
I also had to manually create my bot-config.json file, fortunately it is very small/short as per above.

**Comment by IoALFANO** [comment]
@Malexion you should create the file bot-config.json like that : 
```
{
    "discord": {
        "token": "token-here"
    }
}
````
It will be easier for people to setup the bot.

**Comment by Malexion** [comment]
Yeah I used to have it checked in but then I kept committing my api so I kept having to regenerate them

**Comment by 22tooty2** [comment]
was also stuck here,
this helped. thank you!

i recomend to add this to the "installation" part on the main page.

**Comment by dudeofawesome** [comment]
Just FYI, there's a bug in the default config generation code which prevents it from actually creating the file.

the `writeJson` method is async, but isn't awaited, and then the file (which hasn't started writing to disk yet) and immediately `require`d, throwing an exception and halting execution before the json can be written to disk.

**Comment by Malexion** [comment]
Should be resolved in the latest commit
